endpoint.bal

```diff
@@ -54,10 +56,12 @@ public client class Client {
     # + queueName - Name of the queue to be created 
     # + attributes - Other attribute parameters 
     # + return - If success, URL of the created queue, else returns error
-    public remote function createQueue(string queueName, map<string> attributes)
-            returns @tainted string|OperationError{
-        string amzTarget = AMAZON_SQS_API_VERSION + "." + ACTION_CREATE_QUEUE;
-        string endpoint = "/";
+    @display {label: "Create queue"}
+    remote isolated function createQueue(@display {label: "Queue name"} string queueName, 
+                                @display {label: "Map of attributes"} map<string> attributes)
```

Comment:
`Map of attributes` is not a meaningful name for a display name.

Reply:
@kapilraajP, please follow the SQS API Reference doc and implement all operations according to that. The createQueue action is taking three inputs. But you have implemented only two. Also, you have missed the tag parameter as well. As @abeykoon mentioned, you have to set all not required parameters to optional.

Reply:
Fixed